### PR TITLE
Add /usr/lib to PyTorch manifest

### PR DESCRIPTION
**Pull request by jinengandhi-intel**
PyTorch when installed via apt on Debian12 fails with `libgloo0` import error as /usr/lib path (where the lib and some others are installed) is missing in manifest file. This PR adds `/usr/lib` path to `LD_LIBRARY_PATH`, `fs.mounts` and `sgx.trusted_files`

### CODE REVIEW

**Review by dimakuv — CHANGES_REQUESTED** [review]
> Reviewed 1 of 1 files at r1, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/gramineproject/examples/72)* status: all files reviewed, 6 unresolved discussions, not enough approvals from maintainers (3 more required), not enough approvals from different teams (2 more required, approved so far: ) (waiting on @jinengandhi-intel)

___
*[`-- commits` line 2 at r1](https://reviewable.io/reviews/gramineproject/examples/72#-NaAnTgm1pElJ8LUioYI):*
`/usr/lib` should be in backticks. We can fix it during final rebase.
___
*[`-- commits` line 5 at r1](https://reviewable.io/reviews/gramineproject/examples/72#-NaAnacyDvdnXIiWMT1K):*
I would rephrase to this:
```
On Debian 12, PyTorch installation uses the package python3-torchvision
of version 0.14.1. This version of python3-torchvision depends on the `libgloo.so`
library that is installed under `/usr/lib`. Before this commit, this directory was
missing from the PyTorch manifest file, so we need to add it.
```
___
*[`pytorch/pytorch.manifest.template` line 12 at r1](https://reviewable.io/reviews/gramineproject/examples/72#-NaAogLj3NleIYp9OK30:-NaAogLj3NleIYp9OK31:b-qp5km4) ([raw file](https://github.com/gramineproject/examples/blob/7f419f62a706e582c46bf3fad9b0fe6e44df2b2a/pytorch/pytorch.manifest.template#L12)):*
> ```
> # In Debian12 we install torchvision via apt, which installs python3-torchvision
> # (0.14.1-2) at /usr/lib/ which is missing in the manifest template, causing pytorch
> # to fail with Gramine.
> ```

I would remove this comment, as it is already captured in the commit history, so a curious reader can do git blame and see why `/usr/lib` was added.
___
*[`pytorch/pytorch.manifest.template` line 13 at r1](https://reviewable.io/reviews/gramineproject/examples/72#-NaAosdCAglaPUWfXnSj:-NaAosdCAglaPUWfXnSk:b-5xhbow) ([raw file](https://github.com/gramineproject/examples/blob/7f419f62a706e582c46bf3fad9b0fe6e44df2b2a/pytorch/pytorch.manifest.template#L13)):*
> ```
> # (0.14.1-2) at /usr/lib/ which is missing in the manifest template, causing pytorch
> # to fail with Gramine.
> loader.env.LD_LIBRARY_PATH = "/lib:{{ arch_libdir }}:/usr/{{ arch_libdir }}:/usr/lib"
> ```

The order is not typical now. Please change to:
```
loader.env.LD_LIBRARY_PATH = "/lib:/usr/lib:{{ arch_libdir }}:/usr/{{ arch_libdir }}"
```
___
*[`pytorch/pytorch.manifest.template` line 32 at r1](https://reviewable.io/reviews/gramineproject/examples/72#-NaAozzE574pz_iOtmFU:-NaAozzE574pz_iOtmFV:b-rhhmv1) ([raw file](https://github.com/gramineproject/examples/blob/7f419f62a706e582c46bf3fad9b0fe6e44df2b2a/pytorch/pytorch.manifest.template#L32)):*
> ```
> 
>   { type = "tmpfs", path = "/tmp" },
>   { path = "/usr/lib", uri = "file:/usr/lib" },
> ```

Please move this new entry to right-after `/lib` entry (so, right-after the second entry)
___
*[`pytorch/pytorch.manifest.template` line 54 at r1](https://reviewable.io/reviews/gramineproject/examples/72#-NaAp4ryECYM6GcZl2p1:-NaAp4ryECYM6GcZl2p2:bllm6cy) ([raw file](https://github.com/gramineproject/examples/blob/7f419f62a706e582c46bf3fad9b0fe6e44df2b2a/pytorch/pytorch.manifest.template#L54)):*
> ```
>   "file:input.jpg",
>   "file:alexnet-pretrained.pt",  # Pre-trained model saved as a file
>   "file:/usr/lib/",
> ```

Please move this new entry to right-after `"file:{{ gramine.runtimedir() }}/"` entry (so, right-after the third entry)

**Review by jinengandhi-intel — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/gramineproject/examples/72)* status: 0 of 1 files reviewed, 6 unresolved discussions, not enough approvals from maintainers (3 more required), not enough approvals from different teams (2 more required, approved so far: ) (waiting on @dimakuv)

___
*[`-- commits` line 2 at r1](https://reviewable.io/reviews/gramineproject/examples/72#-NaAnTgm1pElJ8LUioYI):*
<details><summary><i>Previously, dimakuv (Dmitrii Kuvaiskii) wrote…</i></summary><blockquote>

`/usr/lib` should be in backticks. We can fix it during final rebase.
</blockquote></details>

Done.
___
*[`-- commits` line 5 at r1](https://reviewable.io/reviews/gramineproject/examples/72#-NaAnacyDvdnXIiWMT1K):*
<details><summary><i>Previously, dimakuv (Dmitrii Kuvaiskii) wrote…</i></summary><blockquote>

I would rephrase to this:
```
On Debian 12, PyTorch installation uses the package python3-torchvision
of version 0.14.1. This version of python3-torchvision depends on the `libgloo.so`
library that is installed under `/usr/lib`. Before this commit, this directory was
missing from the PyTorch manifest file, so we need to add it.
```
</blockquote></details>

Done.
___
*[`pytorch/pytorch.manifest.template` line 12 at r1](https://reviewable.io/reviews/gramineproject/examples/72#-NaAogLj3NleIYp9OK30:-NaB5z6JEQJm4W0x9PkU:b-896fix) ([raw file](https://github.com/gramineproject/examples/blob/7f419f62a706e582c46bf3fad9b0fe6e44df2b2a/pytorch/pytorch.manifest.template#L12)):*
<details><summary><i>Previously, dimakuv (Dmitrii Kuvaiskii) wrote…</i></summary><blockquote>

I would remove this comment, as it is already captured in the commit history, so a curious reader can do git blame and see why `/usr/lib` was added.
</blockquote></details>

Done.
___
*[`pytorch/pytorch.manifest.template` line 13 at r1](https://reviewable.io/reviews/gramineproject/examples/72#-NaAosdCAglaPUWfXnSj:-NaB6-KA2w4TitbjKmGu:b-896fix) ([raw file](https://github.com/gramineproject/examples/blob/7f419f62a706e582c46bf3fad9b0fe6e44df2b2a/pytorch/pytorch.manifest.template#L13)):*
<details><summary><i>Previously, dimakuv (Dmitrii Kuvaiskii) wrote…</i></summary><blockquote>

The order is not typical now. Please change to:
```
loader.env.LD_LIBRARY_PATH = "/lib:/usr/lib:{{ arch_libdir }}:/usr/{{ arch_libdir }}"
```
</blockquote></details>

Done.
___
*[`pytorch/pytorch.manifest.template` line 32 at r1](https://reviewable.io/reviews/gramineproject/examples/72#-NaAozzE574pz_iOtmFU:-NaB60219SWoSDXrMwug:b-896fix) ([raw file](https://github.com/gramineproject/examples/blob/7f419f62a706e582c46bf3fad9b0fe6e44df2b2a/pytorch/pytorch.manifest.template#L32)):*
<details><summary><i>Previously, dimakuv (Dmitrii Kuvaiskii) wrote…</i></summary><blockquote>

Please move this new entry to right-after `/lib` entry (so, right-after the second entry)
</blockquote></details>

Done.
___
*[`pytorch/pytorch.manifest.template` line 54 at r1](https://reviewable.io/reviews/gramineproject/examples/72#-NaAp4ryECYM6GcZl2p1:-NaB60t82DuqAKu_wuWb:b-896fix) ([raw file](https://github.com/gramineproject/examples/blob/7f419f62a706e582c46bf3fad9b0fe6e44df2b2a/pytorch/pytorch.manifest.template#L54)):*
<details><summary><i>Previously, dimakuv (Dmitrii Kuvaiskii) wrote…</i></summary><blockquote>

Please move this new entry to right-after `"file:{{ gramine.runtimedir() }}/"` entry (so, right-after the third entry)
</blockquote></details>

Done.

**Review by dimakuv — APPROVED** [review]
> Reviewed 1 of 1 files at r2, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/gramineproject/examples/72)* status: all files reviewed, all discussions resolved, not enough approvals from maintainers (2 more required), not enough approvals from different teams (1 more required, approved so far: Intel)

**Review by kailun-qin — APPROVED** [review]
> Reviewed 1 of 1 files at r2, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/gramineproject/examples/72)* status: all files reviewed, all discussions resolved, not enough approvals from maintainers (1 more required), not enough approvals from different teams (1 more required, approved so far: Intel)

**Review by mkow — APPROVED** [review]
> Reviewed 1 of 1 files at r2, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/gramineproject/examples/72)* status: :shipit: complete! all files reviewed, all discussions resolved
